### PR TITLE
fix(gridlist): The gridlist will now lay out everytime a tile is added

### LINKED ISSUE #2227: mdGridList: md-grid-tile elements are gone when the grid list is re-created
First of all, I am using the latest builds from: http://rawgit.com/angular/bower-material/master/angular-material.js and this issue was observed on Chrome v41.

We are dynamically creating mdGridList tiles, and refreshing those tiles from script was working as expected before. However today, when we try to refresh them as before, tiles cannot be seen anymore. I am not sure when this was broken, but I am certain that it was okay on April 7th.

Codepen for the issue: http://codepen.io/ecem/pen/myZVgv, normally it's looking good, but when we click refresh the tiles are gone.

### PR DESCRIPTION
**Pull request by shyndman**
Fixes #2227

### CODE REVIEW

File: src/components/gridList/demoDynamicTiles/index.html [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <div ng-app="gridListDemo" ng-controller="gridListDemoCtrl as vm" flex>
-  <md-button ng-click="vm.rotate()">Rotate</md-button>
```
Comment (ThomasBurleson) [inline src/components/gridList/demoDynamicTiles/index.html]: Why the removal of the 'rotate' feature? Did you consider it too confusing for visitors?

Comment (shyndman) [inline src/components/gridList/demoDynamicTiles/index.html]: Yeah, didn't think it added much to the demo, but if you think otherwise I can add it back.

Comment (ThomasBurleson) [inline src/components/gridList/demoDynamicTiles/index.html]: No strong opinion either way.

Comment (shyndman) [inline src/components/gridList/demoDynamicTiles/index.html]: Cool. Let's leave it out.

**Pull request closed without merging**